biopandas/mmcif/pandas_mmcif.py

```diff
@@ -25,14 +25,15 @@
 
 
 class PandasMmcif:
-    def __init__(self, use_auth: bool = True):
+    def __init__(self, use_auth: bool = True, af2_version: int = 2):
```

Comment:
Would this be for specifying which alphafold database to use? I don't think we need this as an `__init__` attribute though? Can't it be specified in `fetch_mmcif(source="")`? 

E.g., something like

```python
if source == 'alphafold2-v1':
    af2_version = 1
elif source == 'alphafold2-v2':
    af2_version = 2
else:
    ...
```

Reply:
Yep, good call I think it belongs somewhere else. The param specifies which release of the database to retrieve the structure from. I don't anticipate it being changed much - the latest version is what I expect 99% of people would use - it was intended to reduce the maintenance burden in between new releases and updating the default.